packages/Dendrogram/src/viewers/inject-tree-for-grid2.ts

```diff
@@ -115,10 +114,10 @@ export function injectTreeForGridUI2(
   try {
     const lineWidthProperty = DG.Property.int(D_PROPS.lineWidth,
       (obj) => {
-        let k = 11;
+        const k = 11;
```

Comment:
what is `k`?

Reply:
I guess it was used for debugging. I was asked to leave it there as is, my linter changed it to const on auto

Reply:
it's okay that it's const, I just were curious if it really needed to be there or not